js/functions.js

```diff
@@ -31,3 +31,23 @@ const getNumberFromString = (string) => {
 };
 
 getNumberFromString('2023 год');
+
+/**Переводит сроку со временем в количество минут.*/
+const getMinutesNumberFromTimeString = (time) => {
+  const SplitToHoursAndMinutes = time.split(':');
+  const numberOfMinutes = Number(SplitToHoursAndMinutes[0]) * 60 + Number(SplitToHoursAndMinutes[1]);
```

Comment:
1. С большой буквы у вас только классы и енамы
```suggestion
  const splitToHoursAndMinutes = time.split(':');
  const numberOfMinutes = Number(splitToHoursAndMinutes[0]) * 60 + Number(splitToHoursAndMinutes[1]);
```
2. Можно сразу привести к числу через map
```suggestion
  const splitToHoursAndMinutes = time.split(':').map(numberAsString => Number(numberAsString));
  const numberOfMinutes = splitToHoursAndMinutes[0] * 60 + splitToHoursAndMinutes[1];
```
или даже так
```suggestion
  const splitToHoursAndMinutes = time.split(':').map(Number);
  const numberOfMinutes = splitToHoursAndMinutes[0] * 60 + splitToHoursAndMinutes[1];
```
3. Можно деструктуризацией разбить на более понятные имена
```suggestion
  const [hours,  minutes] = time.split(':').map(Number);
  const numberOfMinutes = hours * 60 + minutes;
```

5. `60` магическое число сейчас. Вынеси его в КОНСТАНТУ в начале модуля

Reply:
Все поняла и поправила!